themes/axin/README.md

```diff
@@ -0,0 +1,5 @@
+# Axi-Dark
```

Comment:
```suggestion
# Axin-Dark
```

By the way, what are those `-Dark` in the titles of READMEs? The sample images contain `-dark` as a suffix in the filenames, but it just represents that the image is created using a terminal with a dark background, i.e., `-dark` is not the property of the OMB theme but the property of the terminal theme used to create the sample. Could you remove "Dark" from the headings?

Reply:
If you want to include it in the name, could you explain the reason?

Maybe some themes do not consider the appearance of the light terminal background, but it's not explicit. Also, the theme names `axin`, `bakke`, and `bobby` don't include the additional word "dark" like the theme [`powerline-light`](https://github.com/ohmybash/oh-my-bash/tree/master/themes/powerline-light).